tests/php/ORM/UnionListTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace SilverStripe\ORM\Tests;
+
+use SilverStripe\Dev\Debug;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\UnionList;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\ORM\Tests\DataObjectTest\ValidatedObject;
+
+class UnionListTest extends SapphireTest
+{
+    /**
+     * Borrow the model from DataObjectTest
+     * {@inheritDoc}
+     */
+    protected static $fixture_file = 'DataObjectTest.yml';
```

Comment:
Getting rid of this and 'getExtraDataObjects()' caused by tests to fallover in an odd way:
"42S02-1146: Table 'ss_tmpdb2509246.dataobjecttest_validatedobject' doesn't exist"

Might be a Windows thing or slight configuration issue. Anyway, I've pushed up some of the latest changes. If you spot what I'm doing wrong, that'd be awesome!